semantic-python/src/Prelude.score

```diff
@@ -1,8 +1,22 @@
 {
-  type <- \name -> \bases -> \dict ->
-    #record { __name: name, __bases: bases, __dict: dict };
+  type <- \name -> \super -> \slots ->
+    #record { __name: name, __super: super, __slots: slots };
 
-  object <- type "object" #unit #record{};
+  instance <- \class -> \prim -> \slots ->
+    #record { __super: class, __prim: prim, __slots: slots };
```

Comment:
Instances are also a triple, holding the object’s class, any primitive Core terms associated with it, and a slots record like those in classes.